*** run.test.orig	Thu Sep  1 15:04:44 1994
--- run.test	Mon Oct 23 13:11:49 1995
***************
*** 16,21 ****
--- 16,23 ----
  		DICT=/usr/share/dict/words
  	elif [ -f /usr/dict/words ]; then
  		DICT=/usr/dict/words
+ 	elif [ -f /usr/share/lib/dict/words ]; then
+ 		DICT=/usr/share/lib/dict/words
  	else
  		echo 'run.test: no dictionary'
  		exit 1
***************
*** 537,543 ****
  	awk '{ for (i = 1; i < 20001; ++i) print $0 }' > $TMP1
  	for type in btree; do
  		rm -f $TMP2 $TMP3
! 		for i in `sed 20000q $DICT | rev`; do
  			echo p
  			echo k$i
  			echo d$mdata
--- 539,549 ----
  	awk '{ for (i = 1; i < 20001; ++i) print $0 }' > $TMP1
  	for type in btree; do
  		rm -f $TMP2 $TMP3
! 		for i in `sed 20000q $DICT | \
! 			perl -e 'while (<>) { \
! 				 chop; \
! 				 print STDOUT (scalar(reverse($_)),"\n"); \
! 			}'`; do
  			echo p
  			echo k$i
  			echo d$mdata
